File: ReactNativeWrapperExample_1/ios/Livelike/landscape/LandscapeWODViewManager.m
//
//  LiveLikeiOSViewManager.m
//  ReactLivelike
//
//  Created by Changdeo Jadhav on 16/01/22.
//

#import "React/RCTViewManager.h"
#import <React/RCTBridgeModule.h>
#import <React/RCTEventEmitter.h>
#import <React/RCTEventDispatcher.h>

@interface RCT_EXTERN_MODULE(LandscapeWODViewManager, RCTViewManager)
RCT_EXPORT_VIEW_PROPERTY(programId, NSString)
RCT_EXPORT_VIEW_PROPERTY(toggleWidget, BOOL)
@end

File: ReactNativeWrapperExample_1/ios/Livelike/landscape/LandscapeWODViewManager.swift
//
//  LiveLikeiOSViewManager.swift
//  ReactLivelike
//
//  Created by Changdeo Jadhav on 16/01/22.
//

import Foundation
import UIKit
import React

@objc(LandscapeWODViewManager)
class LandscapeWODViewManager: RCTViewManager {
  override func view() -> UIView! {
    return LandscapeWODView()
  }
  override static func requiresMainQueueSetup() -> Bool {
     return true
   }
}
